### ISSUE TITLE
Doom 3 bfg FPS

### ISSUE THREAD

**Issue by quidamisatis** [glKarin/com.n0n3m4.diii4a#240]
Doom 3 bfg always runs at 120 FPS even when settings are set to 60 FPS, which tends to consume a lot of battery and make the phone heat a lot.

**Comment by glKarin** [comment]
Try change cvar com_engineHz



At 2024-08-02 19:15:12, "quidamisatis" ***@***.***> wrote:

Doom 3 bfg always runs at 120 FPS even when settings are set to 60 FPS, which tends to consume a lot of battery and make the phone heat a lot.

**Comment by quidamisatis** [comment]
I have already tried changing com_engineHz and it doesn't change anything, the game always runs at 120 no matter what value is entered.

**Comment by Deepsea433** [comment]
What phone and settings you using to play BFG? It run very bad on my phone. How can you get 120 fps?!

**Comment by quidamisatis** [comment]
After reading your comment, I tried the game again and I just realized that the game displays an FPS between 80 and 95 with the developer options FPS counter. But since you made me doubt, I tried with TakoStats and indeed, the game runs at 60 FPS with some drops to 30 . So, it's my fault and my initial message is no longer relevant. Xiaomi 12 pro (SD 8gen1)

**Comment by Deepsea433** [comment]
I am using REDMAGIC 8 pro snap 8 gen 2 and the game run at 30 60 fps on low graphics setting, usually 30 40 fps.

**Comment by quidamisatis** [comment]
Maybe it has something to do with where you are in the game because I only tested it in the first few minutes. Perhaps in longer sessions or elsewhere in the game, the FPS results will be surely worse than yours . Also i am playing at 1680×756 resolution, anti-aliasing Off, vertical synchro Off.

**Comment by Deepsea433** [comment]
Yes, i think the BFG port for android just for testing, maybe in the future with snap 8 gen 5 or gen 6 we can play it stable 60fps... By the way Doom 3 original run smooth and the graphics still amazing enough for me, so BFG is not really important... Doom 3 forever!
